tests/unit/aroi/test_aroi_validation.py

```
@@ -551,5 +551,735 @@ def test_fingerprint_sets_populated(self):
+            self.assertGreaterEqual(w['count'], 1)
+
+    def test_aroi_warnings_log_records_schema_mismatch(self):
+        """B6 (final): schema-version mismatch logged in the warning feed."""
+        from allium.lib.aroi_validation import (
+            check_schema_version, get_aroi_warnings_log,
+            reset_aroi_warnings_log,
+        )
+        reset_aroi_warnings_log()
+        check_schema_version(
+            {'metadata': {'aroivalidator_schema_version': 99}},
+            lambda m: None,
```

Ruff (0.15.12)

[warning] Unused lambda argument: `m`

(ARG005)